anyway/parsers/twitter.py

```diff
@@ -0,0 +1,114 @@
+import datetime
+import re
+import os
+
+import pandas as pd
+import tweepy
+
+from .news_flash_classifiers import classify_tweets
+from .location_extraction import geocode_extract, manual_filter_location_of_text, get_db_matching_location, \
+    set_accident_resolution
+
+
+def extract_accident_time(text):
+    """
+    extract accident's time from tweet text
+    :param text: tweet text
+    :return: extracted accident's time
+    """
+    reg_exp = r'בשעה (\d{2}:\d{2})'
+    time_search = re.search(reg_exp, text)
+    if time_search:
+        return time_search.group(1)
+    return None
+
+
+to_hebrew = {
+    'mda_israel': 'מגן דוד אדום'
+}
+
+
+def scrape(screen_name, latest_tweet_id, count=100):
+    """
+    get all user's recent tweets
+    """
+    auth = tweepy.OAuthHandler(os.environ['TWITTER_CONSUMER_KEY'], os.environ['TWITTER_CONSUMER_SECRET'])
+    auth.set_access_token(os.environ['TWITTER_ACCESS_KEY'], os.environ['TWITTER_ACCESS_SECRET'])
+    api = tweepy.API(auth, parser=tweepy.parsers.JSONParser())
+
+    # fetch the last 100 tweets if there are no tweets in the DB
+    if latest_tweet_id == 'no_tweets':
+        all_tweets = api.user_timeline(
+            screen_name=screen_name, count=count, tweet_mode='extended')
+    else:
+        all_tweets = api.user_timeline(
+            screen_name=screen_name, count=count, tweet_mode='extended', since_id=latest_tweet_id)
+                                  # ?? ^^^^^^^^^
```

Comment:
I mean, why would we have a count when there's a lower bound on the oldest tweet?